src/components/input/styled.js

```diff
@@ -81,13 +81,9 @@ export const SuccessIcon = styled(StyledIcon)`
 export const IconContainer = styled(Flex)``
 
 export const MetaContainer = styled.div`
```

Comment:
Maybe change `div` to `Flex` component. 
( I know it's not a part of this PR, just mentioning it )